instrumentation/spring/spring-webflux-5.0/javaagent/src/test/groovy/client/SpringWebFluxSingleConnection.groovy

```diff
@@ -29,7 +31,8 @@ class SpringWebFluxSingleConnection implements SingleConnection {
         clientOptions.poolResources(PoolResources.fixed("pool", 1, HttpClientTest.CONNECT_TIMEOUT_MS))
       })
     } else {
-      def httpClient = reactor.netty.http.client.HttpClient.create(reactor.netty.resources.create("pool", 1)).tcpConfiguration({ tcpClient ->
```

Comment:
I confirmed on `main` that this branch never runs even for latest deps test. No clue why. Also no clue why my PR fixed it (Dr. Murphy probably involved). All the changes in this package were to get real latest deps test to work.